akka-docs/src/main/paradox/java/multi-jvm-testing.md

```diff
@@ -0,0 +1 @@
+../scala/multi-jvm-testing.md
```

Comment:
In the ```## How to Test``` section, there is reference: 

```
@ref:[Multi JVM Testing](multi-jvm-testing.md), 
```

And when paradox generates a page for java, **it tries to look up `/java/multi-jvm-testing.md`** (not `/scala/multi-jvm-testing.md`) even though:

* `/java/cluster-usage.md` is a symlink to `/scala/cluster-usage.md`
* and the reference to `multi-jvm-testing.md` is wrapped in `@@@ div {.group-scala}`